benchmark.go

```diff
@@ -188,12 +213,57 @@ func runBenchmark(out output, logger *zap.Logger, allOpts Options, resolved reso
 		zap.Time("startTime", start),
 	)
 
+    if opts.Format == "json" || opts.Format == "JSON" {
+        // Calling JSON output helper method
```

Comment:
comments are not giving extra useful information. Since the logic is also self-descriptive, consider remove the comments